airflow/contrib/hooks/gcp_pubsub_hook.py

```diff
@@ -12,26 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from uuid import uuid4
+
 from apiclient.discovery import build
 from apiclient import errors
 
 from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
 
 
+def _format_subscription(project, subscription):
+    return 'projects/%s/subscriptions/%s' % (project, subscription)
```

Comment:
nit: format method is preferred for consistency and flexibility, here and other places.